data/json/vehicles/trains.json

```diff
@@ -238,5 +238,16 @@
       { "x": 1, "y": 0, "parts": [ "headlight", "battery_motorbike" ] },
       { "x": -1, "y": 0, "parts": [ "muffler", "box" ] }
     ]
+  },
+  {
+    "id": "miniature_train_loco",
+    "type": "vehicle",
+    "name": "Miniature train locomotive",
+    "blueprint": [ "oo" ],
+    "parts": [
+      { "x": 1, "y": 0, "parts": [ "frame_vertical_2", "engine_steam_small", "rail_wheel_small_pair", "headlight" ] },
```

Comment:
Probably doesn't need to have a functional (electric) headlight if it can't actually supply power to it (or alternatively should have an alternator on engine and small battery to supply said power).

Reply:
Removed headlight